### DISCUSSION TITLE
Winforms - Could Not Load File Or Assembly 'CefSharp.Core.Runtime'

### DISCUSSION THREAD

**Discussion by slane-ck** [cefsharp/CefSharp#4177] [General]
I implemented CefSharp version 103.0.80.0 in my Winforms .net frameword 4.8 app, and it worked beautifully. A couple days later, I started getting the error: Could not load file or assembly 'CefSharp.Core.Runtim, Version=103.0.80.0, Culture=neutral, PublicKeyToken=40c' or one of its dependencies. The system cannot find the file specified. I'm setting the Target Platform to AnyCPU, I'm not sure if that has to do with it. I've followed all of the suggestions in https://github.com/cefsharp/CefSharp/wiki/Frequently-asked-questions#Runtime_dependencies but nothing has worked so far.

I've run the MinimalExample successfully and can't reproduce the error there.

Any suggestions welcome!

**Comment by amaitland** [comment]
Did you pick one of the options from #1714 for AnyCPU support? 

Adding the following before any other call to CefSharp, Program.Main is usually my preference. 

```
CefRuntime.SubscribeAnyCpuAssemblyResolver();
```

**Reply by slane-ck** [reply]
That worked, thank you!
